extensions/bitwarden/src/components/searchVault/utils/useGetUpdatedVaultItem.ts

```diff
@@ -9,8 +11,19 @@ import { Item } from "~/types/vault";
  * Otherwise, it will wait for the value to be retrieved from the vault.
  */
 function useGetUpdatedVaultItem() {
+  const session = useSession();
+  const bitwarden = useBitwarden();
   const getItemFromVault = useVaultItemSubscriber();
 
+  async function getItemNoCache(id: string): Promise<Item> {
+    if (!session.active || !session.token) {
+      throw new Error("No session token available");
+    }
```

Comment:
No need to check this, this is already checked when bootstrapping the session context.

Don't forget to remove the import!

```suggestion
  const bitwarden = useBitwarden();
  const getItemFromVault = useVaultItemSubscriber();

  async function getItemNoCache(id: string): Promise<Item> {
```